### DISCUSSION TITLE
A Question about YouTube Subtitles

### DISCUSSION THREAD

**Discussion by CookieDoodle** [FrostCo/AdvancedProfanityFilter#443] [Q&A]
Did YouTube change how they deal with swear words in their subs again? Now instead of showing it as __ they just don't show the bad word at all? But that makes the extension useless :( (Not complaining at the creator here just YT) If they did change it... Why would they change how their subs work again?? As if a young child (fr ex) will be spared or kept pure by just hearing the swear but not seeing it typed out??? Sorry if this inappropriate for the discussions page, I just wanted to see if anyone had an answer to this frustration of mine. Feel free to delete...

**Comment by richardfrost** [comment]
Thanks for sharing your thoughts @CookieDoodle. I think this is a great place for it. Unfortunately I think you summarized the situation well. YouTube does seem to completely just remove those "vulgar" words from their captions, and it doesn't really make a lot of sense. I have heard that content creators can choose whether or not to have their captions filtered, but they are filtered by default which is what most people have it set to. If there is one or two creators that you follow that are giving you this issue you may try reaching out to them and request they see if they can adjust that setting, but other than that there really isn't much else we can do. It does seem to be related to when the captions were generated, so older videos will likely still have the raw word, or `[ _ ]` which the filter can recognize, but when they just blank them out we don't really have any option to detect them.
